android/src/main/kotlin/com/example/another_location_plugin/LocationHelper.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.another_location_plugin
+
+import com.google.android.gms.location.LocationResult
+import kotlin.math.pow
+import kotlin.math.round
+
+fun LocationResult.convertToString(): String {
+    return StringBuilder().append(AnotherLocationPlugin.LONGITUDE_TAG)
+        .append(Constants.WHITE_SPACE)
+        .append(this.lastLocation?.longitude?.run {
+            this.cutToTwoDecimals(Constants.DECIMAL_AMOUNT)
+        })
+        .append(Constants.TRIPLE_WHITE_SPACE)
+        .append(AnotherLocationPlugin.LATITUDE_TAG)
+        .append(Constants.WHITE_SPACE)
+        .append(this.lastLocation?.latitude?.run {
+            this.cutToTwoDecimals(Constants.DECIMAL_AMOUNT)
+        }).toString()
+}
+
+/*decimals are the amount*/
+fun Double.cutToTwoDecimals(decimals : Int) : Double {
```

Comment:
this method should be renamed or modified, because if you send via parameters the length of the ending decimals, it not always will be 2.
You can make it that always returns 2 decimals, inside the method and remove the parameter, or change the name and leave it with decimal count from the parameter

Reply:
Changed to cutToDecimals with parameters